test-configs.yaml

```diff
@@ -485,11 +485,41 @@ device_types:
     filters:
       - blacklist: *allmodconfig_filter
 
+  fsl-ls1012a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
+
+  fsl-ls1028a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
+
+  fsl-ls1043a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
+
+  fsl-ls1046a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
+
+  fsl-ls1088a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
+
   fsl-ls2088a-rdb:
     mach: freescale
     class: arm64-dtb
     boot_method: uboot
 
+  fsl-lx2160a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
```

Comment:
Baseline results on staging: https://staging.kernelci.org/test/plan/id/5e9f97128ba9ea152b61e03c/
All passing.

Upstream in LAVA:
https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/fsl-lx2160a-rdb.jinja2